components/esp_lvgl_port/include/esp_lvgl_port.h

```diff
@@ -48,6 +48,7 @@ typedef struct {
  * @brief Rotation configuration
  */
 typedef struct {
+    bool sw_rotate;/*!< Use software rotation (slower) */
```

Comment:
I think, that better place for `bool sw_rotate;` is inside flags, in main structure (`lvgl_port_display_cfg_t`).